Bottom: 74be047e42271c016ee923bff3140cc2c4107b64
Top:    e80376e4e66ca9205db4cff276ae41ee58f53d12
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-08-16 10:19:05 -0500

Build ntel whenever any layer changes

Works by adding an extra step to ntel builds that checks the
change sourestamps and checks out changed layers. Potentially this
could detect changes to multiple layers if they fall within the
tree stable timeout.

This really only works for layers in the same group on gitlab, so any
repository under /ntel/.


---
diff --git a/roles/buildbot-master/files/master.cfg b/roles/buildbot-master/files/master.cfg
index 1245dc16..e1608831 100644
--- a/roles/buildbot-master/files/master.cfg
+++ b/roles/buildbot-master/files/master.cfg
@@ -170,7 +170,7 @@ def codebaseGenerator(chdict):
     """
 
     giturl = giturlparse(chdict['repository'])
-    return "%s/%s" % (giturl.owner, giturl.repo)
+    return giturl.owner
 
     # path = urlparse(chdict['revlink']).path.lstrip('/')
 
diff --git a/roles/buildbot-master/files/openembedded.py b/roles/buildbot-master/files/openembedded.py
index 5cbd252a..9166ece0 100755
--- a/roles/buildbot-master/files/openembedded.py
+++ b/roles/buildbot-master/files/openembedded.py
@@ -9,7 +9,7 @@ c = WorkerConfig = {}
 
 
 DEFAULT_BBFLAGS = '-k'
-DEFAULT_CODEBASE = "ntel/setup-scripts"
+DEFAULT_CODEBASE = "ntel"
 
 # Workers
 # The 'workers' list defines the set of recognized buildworkers. Each element is
